frontend/src/searchContext.tsx

```diff
@@ -31,6 +38,8 @@ export type Option = {
   numeric?: boolean;
 };
 
+export const isOption = (x: any): x is Option => 'label' in x && 'value' in x;
```

Comment:
Why do we need `x: any` followed by `x is Option`  here? Can we use a type besides `any`?

Reply:
this is a type guard that we can use to check if any value is of type Option